NBitcoin/Money.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.Globalization;
+using System.Globalization;

```

Comment:
This is not necessary.

Reply:
What is this btw? I see this glitch here and there.

Reply:
it's the BOM being changed accidentally by text editors

Reply:
I don't get why this project i so broken in regards to line endings